cmd/admin/key.go

```diff
@@ -53,13 +87,70 @@ func (a *admin) spkiHashFromPrivateKey(keyFile string) ([]byte, error) {
 	return spkiHash[:], nil
 }
 
-func (a *admin) blockSPKIHash(ctx context.Context, spkiHash []byte, comment string) error {
+func (a *admin) spkiHashesFromFile(filePath string) ([][]byte, error) {
+	file, err := os.Open(filePath)
+	if err != nil {
+		return nil, fmt.Errorf("opening spki hashes file: %w", err)
+	}
+
+	var spkiHashes [][]byte
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		spkiHex := scanner.Text()
+		if spkiHex == "" {
+			continue
+		}
+		spkiHash, err := hex.DecodeString(spkiHex)
+		if err != nil {
+			return nil, fmt.Errorf("decoding hex spki hash %q: %w", spkiHex, err)
+		}
```

Comment:
Let's include a check that the output length is exactly 32 (or equivalently that the input length is exactly 64).